Xamarin.Forms.Platform.iOS/ModalWrapper.cs

```diff
@@ -14,8 +14,8 @@ internal ModalWrapper(IVisualElementRenderer modal)
 			_modal = modal;
 
 			var elementConfiguration = modal.Element as IElementConfiguration<Page>;
-			if (elementConfiguration?.On<PlatformConfiguration.iOS>().ModalPresentationStyle() == PlatformConfiguration.iOSSpecific.UIModalPresentationStyle.FormSheet)
-				ModalPresentationStyle = UIKit.UIModalPresentationStyle.FormSheet;
+			var modalPresentationStyle = elementConfiguration?.On<PlatformConfiguration.iOS>()?.ModalPresentationStyle() ?? PlatformConfiguration.iOSSpecific.UIModalPresentationStyle.FullScreen;
+			ModalPresentationStyle = modalPresentationStyle.ToNativeModalPresentationStyle();
```

Comment:
@samhouts the change here forces *ModalPresentationStyle* to explicitly be set to FullScreen by default where as before *ModalPresentationStyle* would only get set if you set it to *FormSheet*

on iOS12 *ModalPresentationStyle* is set to *FullScreen* by default but on iOS13 it's set to *Automatic* (Which is a new property on iOS13)  

*Automatic* causes the *ModalPresentationStyle* to be FullScreen on ios12 but PageSheet on iOS13

By just setting the *ModalPresentationStyle* ourselves explicitly it will be FullScreen by default on iOS13 which makes it so users can't dismiss the *Modal* and break everything

Reply:
Perfect! That's the blurb I was hoping for. Thanks!